playbooks/ansible/roles/client.prep/templates/test-info.yml.j2

```diff
@@ -8,6 +8,16 @@ exported_sharenames:
   {{ exported_sharenames | to_nice_yaml | indent(2) }}
 
 test_users:
-  {{ test_users | to_nice_yaml | indent(2) }}
+  {%- for name in config.accounts +%}
+    {%- for user_name in config.accounts[name].users +%}
+      {%- set user = config.accounts[name].users[user_name] +%}
+      {%- if user.samba +%}
+  - username: {{ user_name }}
+    uid: {{ user.uid }}
+    password: {{ user.password }}
+    servers: {{ config.nodes | dict2items | selectattr('value.accounts', 'contains', name) | map(attribute='key') | list }}
```

Comment:
Would `nodes` be more appropriate than `servers`?

Reply:
Done